helm/examples/azure/sa/env-sa

```diff
@@ -2,6 +2,12 @@
 SUBSCRIPTION=
 STORAGE_ACCOUNT_NAME=
 SA_RESOURCE_GROUP=
+
+# Provide AKS Resource Group or the parent Resource group and AKS name
+RESOURCE_GROUP=
```

Comment:
Is this needed since we have SA_RESOURCE_GROUP and AKS_RESOURCE_GROUP?

Reply:
Yes, it is needed since AKS has a different resource group which is required when pull the vnet/subnet information